README.md

```diff
@@ -43,55 +40,55 @@
 
-<https://woodpecker-ci.org/>
+See the [Installation Instructions](https://woodpecker-ci.org/docs/administration/getting-started) for more details.
 
-## ✨ Contribute
+## Support
 
-See [Contributing Guide](https://github.com/woodpecker-ci/.github/blob/main/CONTRIBUTING.md)
+You can support the project by becoming a backer on [Open Collective](https://opencollective.com/woodpecker-ci#category-CONTRIBUTE).
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://woodpecker-ci.org/docs/next/development/getting-started#gitpod)
+<a href="https://opencollective.com/woodpecker-ci" target="_blank"><img src="https://opencollective.com/woodpecker-ci/backers.svg?width=890" alt="Open Collective backers"></a>
 
-## 📣 Translate
+## Documentation
 
-We use an own [Weblate](https://weblate.org/en/) instance at [translate.woodpecker-ci.org](https://translate.woodpecker-ci.org).
+<https://woodpecker-ci.org/docs/>
 
-<a href="https://translate.woodpecker-ci.org/engage/woodpecker-ci/">
-  <img src="https://translate.woodpecker-ci.org/widgets/woodpecker-ci/-/ui/multi-blue.svg" alt="Translation status" />
-</a>
+## Translation
 
-## 👋 Who uses Woodpecker?
+We have a self-hosted [Weblate](https://weblate.org/en/) instance at [translate.woodpecker-ci.org](https://translate.woodpecker-ci.org).
 
-Woodpecker is used by [itself](https://ci.woodpecker-ci.org/woodpecker/woodpecker-ci/), multiple well-known companies, organizations like [Codeberg](https://codeberg.org), hobbyists and many others.
+An overview of the current translation state is available at <https://translate.woodpecker-ci.org/projects/woodpecker-ci/#languages>.
 
-Leave a [comment](https://github.com/woodpecker-ci/woodpecker/discussions/2149) if you're using it as well.
+To help with translation, the language-specific files in <https://github.com/woodpecker-ci/woodpecker/tree/main/web/src/assets/locales> can be edited.
 
-Also consider using the topic `WoodpeckerCI` in your repository, so others can learn from your config and use the hashtag `#WoodpeckerCI` when talking about the project on social media!
+## Woodpecker and Codeberg
 
-Here are some places where people mention Woodpecker:
+Woodpecker is used as the main CI/CD engine at [Codeberg](https://codeberg.org), an alternative Git hosting platform with a focus on privacy and free software development.
+While there are people which contribute to both projects, there is no official relationship between both.
 
-- [GitHub](https://github.com/topics/WoodpeckerCI)
-- [Codeberg](https://codeberg.org/explore/repos?q=woodpeckerci&topic=1)
-- [Twitter](https://twitter.com/search?q=%23WoodpeckerCI&src=typed_query)
-- [Fediverse](https://mastodon.social/tags/WoodpeckerCI)
+While Woodpecker hosts the core repos at GitHub (for historical reasons and outreach), the offical plugins are developed in the [woodpecker-plugins organization at Codeberg](https://codeberg.org/woodpecker-plugins).
```

Comment:
What's about [Kaniko plugin](https://woodpecker-ci.org/plugins/Kaniko)?

Reply:
That is good, but the issue is open and there is no kaniko repo on Codeberg.

Reply:
The goal was not too go into too much detail WRT to plugins. Maybe adding https://woodpecker-ci.org/plugins in addition to the org link is more clear?

Reply:
> in addition

Instead, I would say. Until all repos are migrated.
Or two links: github, codeberg.

Reply:
We then forget to update this part again. There are only a few plugins on GH anyway, and linking the overall plugin registry page lets users find all sources.